### PR TITLE
Implement TSLint and fix all code issues

### PR DESCRIPTION
**Pull request by viman**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
   "scripts": {
     "compile": "rm -rf lib && ./node_modules/.bin/tsc -p ./src",
     "test": "jest",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch",
+    "lint": "./node_modules/tslint/bin/tslint -c tslint.json 'src/**/*.ts' 'test/**/*.ts'"
```
Comment (NagRock) [inline package.json]: Maybe we should run this task while building at travis? `.travis.yml`

Comment (viman) [reply package.json]: Added

**Comment by NagRock** [comment]
@viman thanks for contributing! I'll check it and merge but after https://github.com/NagRock/ts-mockito/pull/42

**Comment by NagRock** [comment]
It looks good! Thanks again @viman for this code standardization.

**Comment by NagRock** [comment]
@viman thanks once again, lint just failed travis CI build because of my mistake :)

**Pull request merged**